src/media/class-wpml-page-builders-media-translate-helper.php

```diff
@@ -0,0 +1,85 @@
+<?php
+
+class WPML_Page_Builders_Media_Translate_Helper {
```

Comment:
@strategio I suggest you drop the `_Helper` from the name. A class with the name `Helper` suggest to me that it's a class that doesn't have a clear role.